### ISSUE TITLE
Getting `chdir not set or invalid`

### ISSUE THREAD

**Issue by bfricka** [roadhump/SublimeLinter-eslint#97]
```
SublimeLinter: chdir not set or invalid, using /Users/brian.frichette/Dev/gc.camera/app/js/gc/camera 
SublimeLinter: eslint: CameraApp.js ['/usr/local/bin/eslint', '--format', 'compact', '--stdin', '--stdin-filename', '__RELATIVE_TO_FOLDER__', '@', '--config /Users/brian.frichette/Dev/gc.camera/grunt/eslint/.eslintrc'] 
SublimeLinter: eslint output:
/Users/brian.frichette/Dev/gc.camera/app/js/gc/camera/CameraApp.js: line 73, col 28, Error - Parsing error: Unexpected token *

1 problem 
```

Project settings look like:

```js
{
  "SublimeLinter": {
    "linters": {
      "eslint": {
        "@disable": false,
        "args": [
          "@",
          "--config ${project}/grunt/eslint/.eslintrc"
        ]
      }
    }
  }
}
```

I've tried as many variants of these settings as I could possibly find:
- with and without `@`, `--stin-filename`
- using relative paths `-c grunt/eslint/.eslintrc`
- setting `chdir` in SublimeLinter

With the above settings it's linting files very strangely: Ignoring the .eslintrc file, but also missing new errors. I don't even know how to debug something like that.

Running eslint directly works flawlessly (as does our build): `eslint -c grunt/eslint/.eslintrc ./app/js/gc/camera/CameraApp.js` produces no errors (since we have the `generators` flag set).

Also, copying my `.eslintrc` into my project base directory allows the linter to work, but this is not feasible since we distribute the the file as part of a git submodule amongst all of our projects.

**Comment by ms88privat** [comment]
did you try it this way?
   ````
 "linters": {
            "eslint": {
                "@disable": false,
                "args": [],
                "chdir": "${project}",
                "excludes": []
            }
        },
```

But I think my .eslintrc is ignored too...

**Comment by roadhump** [comment]
It seems for me that `chdir` warning is not a problem, it is a bug of SublimeLinter itself, but it does not affect this plugin, as I guess.

It works for me with this project file: 

```
{
	"folders":
	[
		{
			"path": "."
		}
	],
      "SublimeLinter": {
        "linters": {
          "eslint": {
            "@disable": false,
            "args": [
              "--config", "${project}/unrelated/.eslintrc"
            ]
          }
        }
      }

}
```

`@` is unnecessary and option name and value should be separated.

**Comment by bfricka** [comment]
That works. I can put in a PR to clarify in the README if you want.

Thanks much.

**Comment by ChaimaeCh** [comment]
Hello ! I tried this because I have the same issue but it didn't work, unfortunately. I think it does not recognize that ${project} is a variable, for me... Do I have to d something else so it works ?

**Comment by roadhump** [comment]
@ChaimaeCh Set SublimeLinter to debug mode (Tools - SublimeLinter - Debug Mode) and check console output (CTRL+`) . It should output full eslint command with full path to config. Check it this path is correct.

**Comment by ChaimaeCh** [comment]
I did but nothing changed... So I deleted everything in args and it worked fine... I don't know exactly why but it worked for me :)

Thank you for your reply :+1:

**Comment by bfricka** [comment]
Closing.

**Comment by joeyyang** [comment]
@ms88privat: THANK YOU for your suggestion to run `eslint --init` in your working directory. SublimeLinter wasn't running anything in my working directory and running that fixed it.
